src/Nancy/project.json

```diff
@@ -7,7 +7,7 @@
   "licenseUrl": "",
 
   "dependencies": {
-    "Microsoft.Framework.Runtime.Interfaces": "1.0.0-beta4"
+    "Microsoft.Framework.Runtime.Interfaces": "1.0.0-beta5"

```

Comment:
yep. Could that be why I wasn't able to run the self hosting sample? -It crashed because some service locator thing was `null`. I'll try.